server/impl/impl.go

```diff
@@ -8,36 +8,53 @@ import (
 	"github.com/gorilla/sessions"
 )
 
+// An Implementation provides all server endpoints for the app.
 type Implementation struct {
 	Options struct {
 		Production bool `long:"production" description:"Run in production mode"`
 	}
 
-	sessionStore     *sessions.CookieStore // lazily initialized; use SessionStore() instead!
+	sessionStore     sessions.Store // could be lazily initialized; use SessionStore() instead!
 	sessionStoreInit sync.Once
 }
 
+// NewImplementation returns a new Implementation intended for production,
+// with a sessions.CookieStore as the internal session store.
 func NewImplementation() *Implementation {
```

Comment:
I don't see this being used anywhere. Right now, I'm assuming that this is here to satisfy some interface.

Reply:
It's used for actual server startup; see https://github.com/downToMeet/downToMeet/blob/d2c28f1cf84a3e295ec2ff8cd05ef8f019bf7906/server/restapi/configure_down_to_meet.go#L21.